doc/api/resources/item_add-ons.rst

```diff
@@ -0,0 +1,241 @@
+Item add-ons
+=====
+
+Resource description
+--------------------
+
+With add-ons, you can specify products that can be bought as an addition to this product. For example, if you host a
+conference with a base conference ticket and a number of workshops, you could define the workshops as add-ons to the
+conference ticket. With this configuration, the workshops cannot be bought on their own but only in combination with a
+conference ticket. You can here specify categories of products that can be used as add-ons to this product. You can also
+specify the minimum and maximum number of add-ons of the given category that can or need to be chosen. The user can buy
+every add-on from the category at most once. If an add-on product has multiple variations, only one of them can be bought.
+The add-ons resource contains the following public fields:
+
+.. rst-class:: rest-resource-table
+
+===================================== ========================== =======================================================
+Field                                 Type                       Description
+===================================== ========================== =======================================================
+id                                    integer                    Internal ID of the add-on
+addon_category                        integer                    Internal ID of the item category the add-on can be
+                                                                 chosen from.
+min_count                             integer                    The minimal number of add-ons that need to be chosen.
+max_count                             integer                    The maximal number of add-ons that can be chosen.
+position                              integer                    An integer, used for sorting
+===================================== ========================== =======================================================
+
+
+Endpoints
+---------
+
+.. http:get:: /api/v1/organizers/(organizer)/events/(event)/items/(item)/addons/
+
+   Returns a list of all add-ons for a given item.
+
+   **Example request**:
+
+   .. sourcecode:: http
+
+      GET /api/v1/organizers/bigevents/events/sampleconf/items/11/addons/ HTTP/1.1
+      Host: pretix.eu
+      Accept: application/json, text/javascript
+
+   **Example response**:
+
+   .. sourcecode:: http
+
+      HTTP/1.1 200 OK
+      Vary: Accept
+      Content-Type: application/json
+
+        {
```

Comment:
I think it is confusing if the JSON part of the response is further indented than the headers (goes for all the samples).